docs/administration/configuring-replication/create-replication-endpoints.md

```diff
@@ -48,12 +43,37 @@ To replicate image repositories from one instance of Harbor to another Harbor or
 1. Click **Test Connection**.
 1. When you have successfully tested the connection, click **OK**.
 
-## Managing Registries  
-
-You can list, add, edit and delete registries under **Administration** -> **Registries**. Only registries which are not referenced by any rules can be deleted.  
+## Access ID and Secret Configuration
+
+- AWS ECR adapters should use access keys, not a username and password.The access key should have sufficient permissions, such as storagepermission.
+- Google GCR adapters should use the entire JSON key generated in theservice account. The namespace should start with the project ID.
+- Quay Registry
+   - Supported scope
+      - [Quay.io](https://quay.io) (cloud version) does not allow to create namespaces (organizations) automaticaly, due to Recaptcha enabled on the Quay.io side.
+      - [RedHat Quay](https://www.openshift.com/products/quay) (enterprise on-premises version) is fully supported (tested on v3.2.0)
+      - [Project Quay](https://github.com/quay/quay) (open sourced version) is also supported theoretically, but has not been tested yet.
+   - Authorization
+      - If you are connecting to a registry without authorization, keep Access ID and Access Secret empty.
+      - If you are connecting to a registry with authorization, you don't need to input an Access ID. Harbor uses json_file as the default Access ID. Input your Access Secret in json format, example:
+      ```
+      {
+      "oauth2\_token": "YmQZ1QZENVmOD6v9kENzmfptNVhgBuy5oVl85eGV", // optional
+      "account\_name": "jack",
+      "docker\_cli\_password": "q6NVazikNqIf4coiQ+JvV4iqiCpkNjE0DLX8ZMQuFRbkHk5iMv6/hd4WdV3W3nyX"
+      }
+      ```
+      - `oauth2\_token` is required only if you want to create an organization automatically.
+      - `account\_name` is your login name. Its not recommended to use a robot account because a robot account can not access different organization's repositories.
+      - `docker\_cli\_password` is your cli password. You can generate this on the Quay ui page.
+   - Tip
```

Comment:
Instead of a Tip, we should make this a `<note>`. 

`{{< note >}}
Docker registry manifest schema1 is not supported.
{{< /note >}}`

If this applies to all Quay registries, then we should make it clearer in the note. `Harbor does not support Docker registry manifest schema1 in Quay registries.` or `Quay registries does not support support Docker registry manifest schema1`

Reply:
@a-mccarthy I have changed the tip => note. PTAL, thanks.